doctest/build.gradle

```diff
@@ -85,9 +88,11 @@ task stopPrometheus() {
 }
 
 stopPrometheus.mustRunAfter startPrometheus
-doctest.dependsOn startOpenSearch
+if (DefaultNativePlatform.currentOperatingSystem.isLinux()) {
```

Comment:
Thank you! 
Can you include a comment saying why we don't run prometheus on non-Linux systems.

Reply:
I'll be happy to know why...

Reply:
Fixed in 82942faa8.